bravado/testing/integration_test.py

```diff
@@ -473,6 +494,34 @@ def test_msgpack_support(self, swagger_http_server):
         assert response.headers['Content-Type'] == APP_MSGPACK
         assert unpackb(response.raw_bytes, encoding='utf-8') == API_RESPONSE
 
+    def test_following_redirects(self, swagger_http_server):
+        # the FidoClient doesn't have a way to turn off redirects being followed
+        # so limit this test to the RequestsClient instead
+        if not isinstance(self.http_client, RequestsClient):
+            pytest.skip('Following redirects is only supported in the RequestsClient')
```

Comment:
Can we make this a blacklist instead of a whitelist? [bravado-asyncio](https://github.com/sjaensch/bravado-asyncio) does have a way to turn off following redirects, and I'd like to be able to run the test there too.

Reply:
Done, although it's somewhat clunky due to how coverage and test dependencies are loaded. If there's better suggestions on how to implement blacklisting without having to load the thing being blacklisted, I'm happy to change it.